Gemfile

```diff
@@ -0,0 +1,72 @@
+source 'https://rubygems.org'
```

Comment:
- [ ] Kindly fix this error 
![image](https://user-images.githubusercontent.com/24703171/210438010-158cfa8b-b201-41d0-ac78-00b040863485.png)
when I run your project I get this error and nothing is displayed in the greeting route.
![image](https://user-images.githubusercontent.com/24703171/210438192-191a3362-813d-4329-8ec5-e307306e4912.png)

Reply:
Hello @gemmen29 . Thank you for your review. The URL from my side is working well.